### ISSUE TITLE
Sensor graph: line gap between origin(0) to 5 is differ from remaining margin gap (5-10.10-15 and so on)

### ISSUE THREAD

**Issue by ArpitaSussol** [openmsupply/mobile#3706]
## Describe the bug

 line gap between origin(0) to 5 is differ and is large compared with remaining margin gap (5-10.10-15 and so on)

### To reproduce

1. Go to Vaccine ->Sensor -> CHoose any sensor with logs record
2. Click on sensor details
3. See error. Gap between Origin & 5 degree C is large compared with 5-10 /10-15/15-20 and so on.
![Screenshot_20210309-151050_mSupply-Mobile](https://user-images.githubusercontent.com/58979436/110452833-a6dc6c00-80ed-11eb-8621-2e7e0dcc1355.jpg)

### Expected behaviour

Since there is equal degree celcius interval, gap should be same between each point.

### Proposed Solution

Leave if you don't know how to fix/implement. Edit this issue description and explain here if you know the best path of implementing the fix within the codebase.

### Version and device info

- App version: V7.0.0 rc9
- Tablet model: Lenovo
- OS version: 8.1

**Comment by katherine-sussol** [comment]
This one is interesting! I could reproduce this initially after launching the app and going straight to the sensor details page - but then after a new set of logs was downloaded the graph 'window' updated to not show the lines at all 🤔  

Before download:
![reproduced](https://user-images.githubusercontent.com/65875762/110569680-af01dd80-81b9-11eb-8f13-ce3c2657a1ad.png)

After download:
![updated](https://user-images.githubusercontent.com/65875762/110569897-0011d180-81ba-11eb-9bf5-41c1f3e444e2.png)

**Comment by joshxg** [comment]
@katherine-sussol Just realized- the threshold lines are for temperatures which aren't on the graph- oops!

**Comment by katherine-sussol** [comment]
Yeah, possibly a bit unrealistic because of such a low threshold vs the actual temperature 🤣 , although still not sure why sometimes the min/max graph domain window accounts for the breach configs, and sometimes it ignores them and only uses the temperatures to calculate the window 🤔
